.github/workflows/publish-to-ecr.yaml

```diff
@@ -0,0 +1,70 @@
+name: Publish image to ECR
+
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: Repository environment name
+        default: 'jobbergate-api-staging'
+        required: true
+      region:
+        description: AWS region to which upload the package
+        required: true
+        default: 'us-west-2'
+
+jobs:
+  publish:
+    name: Publish to ECR
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Fail if ref is not a tag
+        if: github.ref_type != 'tag'
+        run: |
+          echo "Publish only supported from tag refs!"
+          echo "Got ref_type=${{ github.ref_type }} instead"
+          exit 1
+
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.8
+
+      - uses: Gr1N/setup-poetry@v7
+
+      - id: poetry-package-version
+        name: Get version of project from poetry
+        working-directory: jobbergate-api
+        run: |
+          echo "Poetry version is: $(poetry version --short)"
+          echo "::set-output name=version::$(poetry version --short)"
+
+      - name: Fail if poetry package version doesn't match tag
+        if: ${{ github.ref_name != steps.poetry-package-version.outputs.version }}
+        run: |
+          echo "Poetry package version doesn't match tag!"
+          echo "tag=${{ github.ref_name }}, version=${{ steps.poetry-package-version.outputs.version }}"
+          exit 1
```

Comment:
do you mind removing this? It makes harder to publish alpha and beta versions since it'd require unsignificant commits

Reply:
but if you prefer to keep the way it is, np as well

Reply:
We agreed to keep this test after further discussion.